contribs/gmf/src/services/themesservice.js

```diff
@@ -212,20 +204,18 @@ gmf.Themes.prototype.getThemesObject = function() {
  * @export
  */
 gmf.Themes.prototype.loadThemes = function(opt_roleId) {
-  /**
-   * @param {angular.$http.Response} resp Ajax response.
-   * @return {Object} The "themes" web service response.
-   */
-  var promiseSuccessFn = function(resp) {
-    this.dispatchEvent(gmf.ThemesEventType.LOAD);
-    return /** @type {gmf.ThemesResponse} */ (resp.data);
-  }.bind(this);
 
-  this.promise_ = this.$http_.get(this.treeUrl_, {
+  var deferred = this.deferred_;
+
+  this.$http_.get(this.treeUrl_, {
```

Comment:
Bad things ...
That's probably an argument to move the theme loading in the constructor